lib/emque/producing/message/message.rb

```diff
@@ -36,6 +36,16 @@ def read_raise_on_failure
           end
         end
 
+        def ignored_exceptions(*ignored_exceptions)
+          @ignored_exceptions = ignored_exceptions
+        end
+
+        def read_ignored_exceptions
+          (Array(@ignored_exceptions) +
+            [InvalidMessageError, MessagesNotSentError] +
```

Comment:
I don't think these should be here. If I'm reading the code correctly, this will result in these exceptions always being ignored, regardless of app settings. I think we should let people add these on their own if they don't want them to raise.